eks-deployments/webhook/patch-webhook-template.ps1

```diff
@@ -48,12 +48,24 @@ $kubeConfig = ConvertFrom-Json -InputObject $ret
 $clusterConfig = $kubeConfig.clusters[0].cluster
 $CaBundle = $clusterConfig."certificate-authority-data"
 
+# Get the Server's Kubernetes Version
+$cmd = 'kubectl version --short=true -o json'
+[string]$ret = Invoke-Expression -Command $cmd
+$fullVersion = ConvertFrom-Json -InputObject $ret
+$serverVersion = $fullVersion.serverVersion.minor
+
+$webhookVersion = 'v1.14'
+if ($serverVersion -ge 16) {
```

Comment:
Should we just default to latest if version is not v1.14?

Reply:
I agree. This would be consistent with the k8s deploy-gmsa-webhook script too - https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh#L82